pantry/src/pantry.rs

```diff
@@ -171,19 +184,26 @@ impl PantryEntry {
                         .send()
                 }
             })
-            .await?;
+            .await
+            .map_err(|e| CrucibleError::GenericError(e.to_string()))?;
 
             let content_length = response
                 .headers()
                 .get(reqwest::header::CONTENT_LENGTH)
                 .ok_or("no content length!")
                 .map_err(|e| anyhow!(e))?;
```

Comment:
Will this ? turn into a `CrucibleError`?  Do we care if it does not?

Reply:
It will turn into a `CrucibleError::GenericError`, yeah, which will get returned by the Pantry as a 500.